src/common/log/logHelper.ts

```diff
@@ -75,15 +75,14 @@ export class LogHelper {
         }
 
         switch (error.errorLevel) {
-            case InternalErrorLevel.Error: {
+            case InternalErrorLevel.Error:
                 // Encode the error code to a four-char code - ex, 0198
                 let errorCodeString = (LogHelper.ERROR_CODE_WIDTH + error.errorCode).slice(-LogHelper.ERROR_CODE_WIDTH.length);
```

Comment:
It is deliberate - tslint would complain - it checks for break or return before case statements.